core/templates/domain/objects/FractionModel.ts

```diff
@@ -13,13 +13,10 @@
 // limitations under the License.
```

Comment:
the name for this file should be `fraction.model.ts` according to the convention

Reply:
Done